### PR TITLE
feat: Review page Webhook notifications DOCS-343

### PR DESCRIPTION
**Pull request by nicklem**
Improves the Webhook notifications page.

### :eyes: Live preview
https://feat-review-page-webhook-notificati--docs-codacy.netlify.app/repositories-configure/integrations/webhook-notifications/

### ❗ Reviewers
I suggest comparing it with the old page. The diff will be of little use as there are substantial updates.
https://docs.codacy.com/repositories-configure/integrations/webhook-notifications/

### :construction: To do
- [x] Peer review

### CODE REVIEW

File: docs/repositories-configure/integrations/post-commit-hooks.md [hunk 1]
```diff
@@ -15,8 +15,6 @@ If the switch isn't visible, go to the **Integrations** tab and add the GitHub o
 !!! important
     Make sure that you enable the integration after adding it.
 
-![Adding an integration](images/webhook-integration-add.png)
```
Comment (prcr) [inline docs/repositories-configure/integrations/post-commit-hooks.md:18]: I'd feel more comfortable if we didn't modify this other page in the scope of DOCS-343.

I noticed that you used the file name `webhook-integration-add.png` for the updated screenshot on the page "Webhook notifications", which makes sense. :+1: But I'd just rename this screenshot to a more sensible file name (for example, `post-commit-hook-add.png`) and keep it for now, as we'll have the opportunity to completely review the page "Post-commit hooks" in [DOCS-256](https://codacy.atlassian.net/browse/DOCS-256).

**Comment by prcr** [comment]
[This is the GIF](https://github.com/codacy/docs/pull/1626/files#diff-a2bf063ae6a325b99fc40babf800f1fc99d8baee605747c4f7fb739707839230R30) that's still showing up on the page :nerd_face:

File: docs/repositories-configure/integrations/post-commit-hooks.md (hunk 1 above)
Comment (prcr) [reply docs/repositories-configure/integrations/post-commit-hooks.md:18]: By the way, [this is the GIF](https://github.com/codacy/docs/pull/1626/files#diff-a2bf063ae6a325b99fc40babf800f1fc99d8baee605747c4f7fb739707839230R30) that's still showing up on the "Post-commit hooks" page :nerd_face: We can also take care of that in DOCS-256. :wink:

Comment (nicklem) [reply docs/repositories-configure/integrations/post-commit-hooks.md:18]: Will do, reverting this one.

**Pull request merged**